addons/hw_drivers/iot_handlers/interfaces/PrinterInterface.py

```diff
@@ -52,9 +52,9 @@ def get_devices(self):
 
     def get_identifier(self, path):
         if 'uuid=' in path:
-            identifier = sub('[^a-zA-Z0-9_]', '', path.split('uuid=')[1])
+            identifier = sub('[^a-zA-Z0-9_-]', '', path.split('uuid=')[1])
```

Comment:
may be a good occasion to extract this string into a constant prior to the `if` rather than this copy paste ?